tests/dynamic_data/publications/layer_by_used_servers/layer_patch_without_data_test.py

```diff
@@ -69,6 +69,13 @@ class TestLayer(base_test.TestSingleRestPublication):
     def before_class(self):
         process_client.ensure_reserved_username(self.workspace, headers=AUTHN_HEADERS)
 
+    @pytest.fixture(scope='function', autouse=True)
+    def post_before_patch(self, request):
```

Comment:
Change post_before_patch in base class. E.g. set post_before_patch_scope='function' on class by default, use 'class' in this class.